docs/src/reference/asciidoc/core/kerberos.adoc

```diff
@@ -0,0 +1,484 @@
+[[kerberos]]
+== {krb}
+
+added[6.7]
+
+Securing Hadoop means using {krb}. As of version 6.4, {es} supports {krb} as an authentication method. While the use of
+{krb} is not required for securing {es}, it is a convenient option for those who already deploy {krb} to secure their
+Hadoop clusters. This chapter aims to explain the steps needed to set up {eh} to use {krb} authentication for {es}.
+
+{ehtm} communicates with {es} entirely over HTTP. In order to support {krb} authentication over HTTP, {eh} uses
+the https://tools.ietf.org/html/rfc4178[Simple and Protected GSSAPI Negotiation Mechanism (SPNEGO)] to negotiate which
+underlying authentication method to use (in this case, {krb}) and to transmit the agreed upon credentials to the server.
+This authentication mechanism is performed using the https://tools.ietf.org/html/rfc4559[HTTP Negotiate] authentication
+standard, where a request is sent to the server and a response is received back with a payload that further advances
+the negotiation. Once the negotiation between the client and server is complete, the request is accepted and a
+successful response is returned.
+
+{ehtm} makes use of Hadoop's user management processes; The {krb} credentials of the current Hadoop user are used when
+authenticating to {es}. This means that {krb} authentication in Hadoop must be enabled in order for {eh} to obtain a
+user's {krb} credentials. In the case of using an integration that does not depend on Hadoop's runtime (e.g. Storm),
+additional steps may be required to ensure that a running process has {krb} credentials available for authentication.
+It is recommended that you consult the documentation of each framework that you are using on how to configure security.
+
+[[kerberos-settings]]
+=== Setting up your environment
+
+NOTE: This documentation assumes that you have already provisioned a Hadoop cluster with {krb} authentication enabled.
```

Comment:
`NOTE: This documentation assumes that you have already provisioned a Hadoop cluster with {krb} authentication enabled.` >> `NOTE: This documentation assumes that you have already provisioned a Hadoop cluster with {krb} authentication enabled (required).`